src/MainWindow.cpp

```diff
@@ -793,8 +794,10 @@ void MainWindow::doubleClickTable(const QModelIndex& index)
     if(!index.isValid())
         return;
 
-    // Don't allow editing of other objects than tables
-    bool allowEditing = db.getObjectByName(ui->comboBrowseTable->currentText()).gettype() == "table";
+	// Don't allow editing of other objects than tables (on the browse table)
+
+	bool allowEditing = (m_currentTabTableModel == m_browseTableModel) && (db.getObjectByName(ui->comboBrowseTable->currentText()).gettype() == "table");
+
```

Comment:
Replace the tabs here by four spaces. Personally, I prefer tabs, too, but because the rest of the entire project uses spaces...